scripts/generate_overview.py

```diff
@@ -1,9 +1,9 @@
 import os
 
-overview = open("overview.md", "w")
+overview = open("Overview.md", "w")
```

Comment:
Why capital "O" in Overview.md? Is that so it sorts first?

Reply:
Just an aesthetic choice! Thought it might be a little more appealing for a user to click on and check out. Again, something I am very happy to change back (especially if there is a convention for .md files to be lowercase, not sure if there is).

Reply:
It also does change the sorting order I believe due to the capitalization, but I also reorganized a bunch of the files so there are only a few at the src level, so the README should still be very easy to find.